### PR TITLE
Created add hack route /addhack

### PR DESCRIPTION
**Pull request by HuangStanley050**

### CODE REVIEW

File: handler.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    })
+  };
+};
+
+const message = ({ time, ...rest }) =>
+  new Promise((resolve, reject) =>
+    setTimeout(() => {
+      resolve(`${rest.copy} (with a delay)`);
+    }, time * 1000)
+  );
```
Comment (danieltongaus) [inline handler.js]: Is this file needed?

Comment (HuangStanley050) [reply handler.js]: deleted hanlder.js from the boiler plate and re pushed to this branch

**Review by danieltongaus — APPROVED** [review]
Looks good
